### ISSUE TITLE
Remove need for the MAME menu!??

### ISSUE THREAD

**Issue by ToddLa** [yoshisuga/MAME4iOS#186]
This is a inspirational goal....

We have access to the root MAME select game menu, and to the in-game MAME/configure menu.

....but while we will always keep these as options, the question is why do users need to use the MAME menu and can we do something to make it a easier/better experience??

Some obvious ones....
* controller input mapping.
* video options like bezels
* video options like side-to-side and cocktail.

Can we move some of these up to a native GUI? What have I missed?

Why DO YOU use the MAME menu or MAME UI?

**Comment by Tomasz-Kluczkowski** [comment]
I use The original mame menu mainly for 2 things:
- input mapping (only in rotary joystick games so far - Midnight Resistance for example. As they are otherwise unplayable on an iphone). This also includes the analog sensitivity etc...
- Difficultly/other dipswitches settings
- Sometimes cheats on a lazy day

**Comment by mrjschulte** [comment]
Use it for a bunch of cases as outlined in your original post and Tomasz' follow up.

**Comment by rslkr** [comment]
All above and
- Soft Reset
- Service (Access to Game Settings)
- Slider Controls

**Comment by ToddLa** [comment]
Sounds like the only/main feature we should have inside M4i (in a perfect world) is some sort of input mapping.

**Comment by mrjschulte** [comment]
Yes, input mapping seems to be the main driver for keeping the Ancient MAME Menu around. Does the new Controller remapping in iOS 14+ help us with this in any way @ToddLa ?

**Comment by ToddLa** [comment]
No the iOS mapping is per-app, we would need a mapping UI per-romset
